packages/table/src/TableContainer/TableContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { forwardRef, PropsWithChildren } from 'react';
+
+import { css, cx } from '@leafygreen-ui/emotion';
+
+import TableContainerProps from './TableContainer.types';
+
+const TableContainer = forwardRef<HTMLDivElement, TableContainerProps>(
```

Comment:
I think it would make sense for the Table component to render a `<table>`. This component is also only needed for sticky headers and virtual scrolling - the basic use case without RT doesn't need this

Reply:
I feel strongly that we should bake this in. We uphold this pattern with our input components too, and that's fine. It's undo burden on the consumer to know when/when not to use the TableContainer.